job-service-db/src/main/resources/v3.1.0/changelog.xml

```diff
@@ -24,7 +24,7 @@
                                        http://www.liquibase.org/xml/ns/dbchangelog
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
 
-    <changeSet id="update_procedures_fix_parallel_job_creation" author="Gamma Team">
+    <changeSet id="update_procedures_fix_parallel_job_creation" author="Gamma Team" runOnChange="true">
```

Comment:
The `runOnChange=true` here is required as a previous changelog (2.7.0) was already configured with this setting, meaning that the updated stored procedure script will have already been run at this point breaking the checksum of the 3.1.0 changes.